pkg/util/utils_supported.go

```diff
@@ -60,6 +60,36 @@ func GetRootlessRuntimeDir() (string, error) {
 	return rootlessRuntimeDir, nil
 }
 
+// GetRootlessConfigHomeDir returns the config home directory when running as non root
+func GetRootlessConfigHomeDir() (string, error) {
+	var rootlessConfigHomeDirError error
+
+	rootlessConfigHomeDirOnce.Do(func() {
+		home := os.Getenv("HOME")
+		resolvedHome, err := filepath.EvalSymlinks(home)
+		if err != nil {
+			rootlessConfigHomeDirError = errors.Wrapf(err, "cannot resolve %s", home)
+			return
+		}
+
+		cfgHomeDir := os.Getenv("XDG_CONFIG_HOME")
+		if cfgHomeDir == "" {
+			tmpDir := filepath.Join(resolvedHome, ".config")
+			os.MkdirAll(tmpDir, 0700)
+			st, err := os.Stat(tmpDir)
```

Comment:
Same thought above re: `$HOME` not being created explicitly.

Reply:
see above